### PR TITLE
handle notification

### PR DESCRIPTION
**Pull request by nguyenvanhieu97**
https://edu-redmine.framgia.vn/issues/3510

### CODE REVIEW

**Comment by nguyenvanhieu97** [comment]
ready

File: app/src/main/java/com/cris/nvh/framgiaproject/mediaplayer/MediaPlayerManager.java [hunk 1]
```diff
@@ -211,6 +207,10 @@ public MediaPlayerManager setState(int state) {
 		return this;
 	}
 
+	public void setMediaPlayer(MediaPlayer mediaPlayer) {
+		mMediaPlayer = mediaPlayer;
+	}
+
 	private int ramdomSong() {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/cris/nvh/framgiaproject/mediaplayer/MediaPlayerManager.java]: rename

File: app/src/main/java/com/cris/nvh/framgiaproject/screen/home/HomeFragment.java [hunk 2]
```diff
@@ -168,17 +172,17 @@ private void initImageSlide() {
 
 	private void initData() {
 		if (getArguments() != null) {
-			if (mTracks == null)
-				mTracks = getLocalTracks();
+//			if (mTracks == null)
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/cris/nvh/framgiaproject/screen/home/HomeFragment.java]: 175 176 remove

File: app/src/main/java/com/cris/nvh/framgiaproject/screen/home/HomeFragment.java [hunk 3]
```diff
@@ -200,6 +204,7 @@ public void onServiceConnected(ComponentName componentName, IBinder iBinder) {
 				PlayMusicService.LocalBinder binder = (PlayMusicService.LocalBinder) iBinder;
 				mService = binder.getService();
 				sMiniMediaPlayer.setService(mService);
+//				sMiniMediaPlayer.update();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/cris/nvh/framgiaproject/screen/home/HomeFragment.java]: remove. code trên develop là code sạch

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by nguyenvanhieu97** [comment]
ready

**Pull request merged**